Sources/SFSafeSymbols/SFSymbolExtension.swift

```diff
@@ -2,23 +2,28 @@ import UIKit
 
 @available(iOS 13.0, *)
 public extension SFSymbol {
-    /// Retrieves the corresponding UIImage.
+    
+    /// Retrieves the corresponding `UIImage`.
+    ///
+    /// - Returns: The corresponding `UIImage`.
     @available(iOS 13.0, *)
     var toImage: UIImage {
         UIImage(systemSymbol: self)
     }
 
-    /// Retrieves the corresponding UIImage with the given configuration.
+    /// Retrieves the corresponding `UIImage` with the given configuration.
     ///
-    /// - withConfiguration: The UIImage.Configuration applied to this system image.
+    /// - Parameter withConfiguration: The `UIImage.Configuration` applied to this system image.
```

Comment:
Would you mind using the param name instead of the argument name here and at some other places to follow the same style everywhere? ;)